.github/workflows/ci.yml

```diff
@@ -19,6 +19,8 @@ jobs:
 
     - name: Install
       run: |
+        # This is added by default, and it is often broken, but we don't need anything from it
+        rm -f /etc/apt/sources.list.d/microsoft-prod.list
```

Comment:
Either this will fail or `sudo` is not needed below.

Reply:
whops, copypasta